dockerfiles/extract_diffs/Dockerfile

```diff
@@ -31,11 +28,10 @@ RUN apk update \
         && apk upgrade \
         && apk add --no-cache \
         ca-certificates \
-	    bash \
+        bash \
+        postgresql-client \
         && update-ca-certificates 2>/dev/null || true
 
-ARG CONFIG_FILE=environments/extractDiffs.toml
```

Comment:
Curious why we're removing this config file arg?

Reply:
We aren't - it's also on line 42. This one is a duplicate.